engine/battle/core.asm

```diff
@@ -3826,7 +3827,8 @@ TryToRunAwayFromBattle:
 	cp BATTLEACTION_FORFEIT
 	ld a, DRAW
 	jr z, .fled
-	dec a ; LOSE
+	assert LOSE + 1 == DRAW
+	dec a
```

Comment:
I doubt WIN/LOSE/DRAW will be reordered.. I would expect other issues if someone tried.

Reply:
Also since this operation is `dec`, I think `assert DRAW - 1 == LOSE` would be more obviously related if we even keep this `assert`. (This is the only place it occurs, so I think it's fine to have one.)

Reply:
You have a point.